intel_extension_for_transformers/llm/runtime/graph/models/bloom/bloom.cpp

```diff
@@ -218,17 +223,23 @@ static bool bloom_model_eval_internal(model_context& lctx, const model_token* to
                     cur);
             cur = ne_add(ctx0, ne_repeat(ctx0, model.layers[il].ffn[1], cur), cur);
         }
-
-        cur = ne_mul_mat(ctx0,
-                model.layers[il].ffn[2],
-                cur);
-        cur = ne_add(ctx0, ne_repeat(ctx0, model.layers[il].ffn[3], cur), cur);
-
-        cur = ne_gelu(ctx0, cur);
-
-        cur = ne_mul_mat(ctx0,
-                model.layers[il].ffn[4],
-                cur);
+        if (jblas_fusion_FFN_Add_GeLu_f32f32_support(model.layers[il].ffn[0]->data, model.layers[il].ffn[2]->data,
```

Comment:
Please check input parameters of  jblas_fusion_FFN_Add_GeLu_f32f32_support for Bloom and GPTNeox. Their FFN[0] has different meanings.